token/bearer.go

```diff
@@ -35,6 +36,10 @@ func (b *BearerToken) ToV2() *acl.BearerToken {
 	return &b.token
 }
 
+func (b BearerToken) Empty() bool {
```

Comment:
@alexvanin, @fyrchik, @cthulhu-rider, also, please, take a look at the new `Empty()` method defined on `BearerToken`

this was done to make it possible to know if a token is empty (does not have signature and body) and does trying to verify it [make sense](https://github.com/nspcc-dev/neofs-node/blob/v0.27.4/pkg/services/object/acl/acl.go#L716) in applications. the key reason for such a solution is that `BearerToken` is a wrapper over `api-go` structure, not the alias (as it is done in `Checksum`, `NetworkInfo`, `Signature`, etc) and it is impossible to convert  `nil` v2 token to `nil` SDK token via `NewBearerTokenFromV2` constructor. that fact lead to some bugs when I did changes in `neofs-node` repo since `token == nil` behaviour has changed

perhaps, we need to change wrapping to aliasing strategy in `token` pkg

btw, is there any rule in SDK repo when we prefer 
```go
type SDKStruct {
    v api.Struct // the only one field here
}
``` 
to 
```go
type SDKStruct api.Struct
```
?

Reply:
> What prevents us from implementing nil -> nil construction?

technically nothing but:
1. I tried not to change the behavior as much as possible. I can adapt changes in the node repo, but know nothing about its usage in the service apps
2. such an approach was used in the other pkgs ([container](https://github.com/nspcc-dev/neofs-sdk-go/blob/master/container/container.go#L76), [record](https://github.com/nspcc-dev/neofs-sdk-go/blob/596774ce5bd3be768a00cf9056ae7258d56e9300/eacl/record.go#L211), [table](https://github.com/nspcc-dev/neofs-sdk-go/blob/master/eacl/table.go#L131), etc). I prefer to keep everything in sync: change it everywhere or do that as it is done in all the other places. I think the answer is somewhere near the answer on the question above:
> btw, is there any rule in SDK repo when we prefer
>```
> type SDKStruct {
>     v api.Struct // the only one field here
> }
> ```
> to
> 
> type `SDKStruct api.Struct`
> 
> ?

Reply:
> btw, is there any rule in SDK repo when we prefer

Not sure we have any rules on that. Maybe it makes more sense to use latter for the simple data types, like we done for basic ACL which is basically a number. For the structs first option seems a bit more appealing for me, but again, don't see exact pros or cons.